src/main/java/by/soft/testProject/questionnairePortal/config/SecurityConfig.java

```diff
@@ -4,7 +4,6 @@
 import by.soft.testProject.questionnairePortal.config.security.jwt.JwtTokenProvider;
```

Comment:
UserServiceImpl -> updateById
```
 User updateUser = userRepository.findById(id).orElse(null);

        if (updateUser == null) {
            throw new ServiceException("No user with (" + id + ") id");
        }
```
Can be rewrited to
```
User updateUser = userRepository.findById(id).orElseThrow(() -> new ServiceException("No user with (" + id + ") id"));
```
Related to all optionals in project